sklearn/tests/__init__.py

```diff
@@ -0,0 +1,4 @@
+'''
+This __init__.py file is needed for the tests in this directory to get installed into site-packages, so that executing `nosetests sklearn` outside the source tree will

```

Comment:
Your line is way too long for PEP8. Invest the time to install PEP8 checks in your favourite editor to see these kind of problems while you are editing the code.